src/Business Foundation/App/NoSeriesCopilot/src/Copilot/NoSeriesExt.PageExt.al

```diff
@@ -0,0 +1,60 @@
+pageextension 324 "No. Series Ext" extends "No. Series"
```

Comment:
`Ext` could be a valid AppSource affix of a BC partner.
We should avoid three character affixes on Extension Objects to avoid collision with object names.

Reply:
Ext has not been registered by itself but is part of some longer affixes. Now that we have namespaces though the risk of conflicts is negligeable.